test/integration/controllers/cooperation.spec.js

```diff
@@ -54,12 +55,25 @@ const testOfferData = {
   FAQ: [{ question: 'Do you enjoy being a director of the Hogwarts?', answer: 'Actually yes, i really like it.' }]
 }
 
+const testActiveQuizData = {
+  title: 'My test quiz',
+  items: [
+    {
+      question: 'What is your name?',
+      answers: [
+        { text: '2', correct: true },
```

Comment:
`isCorrect` instead of `correct`

Reply:
In this pull request this field in model is not changed because it is in another pull request which is open now, so after merge it will be fixed